app/platform/platform.c

```diff
@@ -318,6 +317,7 @@ static void pwms_init()
     pwms_duty[i] = DUTY(0);
   }
   pwm_init(500, NULL);
+  get_pin_map();
```

Comment:
It appears to be safe to call there. However, as @TerryE notes, it is symptomatic of a larger mess....